src/lib/storage/local/local-storage.js

```diff
@@ -688,9 +688,9 @@ class LocalStorage {
                   'bugs': version.bugs,
                   'license': version.license,
                   'time': {
-                    modified: item.time ? new Date(item.time).toISOString() : undefined,
+                    modified: item.time ? new Date(item.time).toISOString() : stats.mtime,
                   },
-                  'versions': {},
+                  'versions': {[latest]: "latest"},
```

Comment:
No double quotes, run `npm run lint` you will see the errors in your terminal.

````
/home/ubuntu/verdaccio/src/lib/storage/local/local-storage.js
  693:42  error  Strings must use singlequote  quotes

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.
```

Reply:
Sorry, updated my PR.